build/wdqs-proxy/README.md

```diff
@@ -167,10 +164,9 @@ Hooking into the internal filesystem can extend the functionality of this image.
 | `/etc/nginx/conf.d/wdqs.template` | Template for the nginx config (substituted to `/etc/nginx/conf.d/default.conf` at runtime)                |
 | `/etc/nginx/conf.d/default.conf`  | nginx config. To override this you must also use a custom entrypoint to avoid the file being overwritten. |
 
-
 ## Source
 
-This image is built from this [Dockerfile](https://github.com/wmde/wikibase-release-pipeline/blob/main/build/WDQS/Dockerfile).
+This image is built from this [Dockerfile](https://github.com/wmde/wikibase-release-pipeline/blob/main/build/wdqs/Dockerfile).
```

Comment:
```suggestion
This image is built from this [Dockerfile](https://github.com/wmde/wikibase-release-pipeline/blob/main/build/wdqs-proxy/Dockerfile).
```

Reply:
This is changed in what is currently up here.

Reply:
Sorry, my bad. I looked at the wrong file. It IS resolved. Thanks for doing that!